src/game/Tactical/Weapons.cc

```diff
@@ -797,7 +797,7 @@ static BOOLEAN UseGun(SOLDIERTYPE* pSoldier, INT16 sTargetGridNo)
 	}
 	else
 	{
-		uiHitChance = CalcChanceToHitGun( pSoldier, sTargetGridNo, pSoldier->bAimTime, pSoldier->bAimShotLocation );
+		uiHitChance = CalcChanceToHitGun( pSoldier, sTargetGridNo, pSoldier->bAimTime, pSoldier->bAimShotLocation, true );
```

Comment:
These changes fix the erroneus degradation of attachments and should be a separate commit. We could include it sooner, since the circle stuff still needs the roof fix.

Reply:
accurate aim circle has now been fixed but there is another issue. Shots aimed at the head that should hit will always miss depending on cursor position or something. SoldierToSoldierBodyPartChanceToGetThrough and CalcChanceToHitGun will report 100 chance-to-hit and therefore so will the mod. The actual bullet looks like it is hitting the level below. The result is identical with the mod disabled. This could be separated in the next update.

Reply:
If you can easily reproduce it, definitely report it as a separate issue.